phpunit.xml.dist

```diff
@@ -77,7 +77,7 @@
         <env name="TESTS_ZEND_LDAP_ACCOUNT_DOMAIN_NAME_SHORT" value="EXAMPLE" />
         <env name="TESTS_ZEND_LDAP_ALT_USERNAME" value="anotherUser" />
         <env name="TESTS_ZEND_LDAP_ALT_DN" value="CN=Another User,OU=Sales,DC=example,DC=com" />
-        <env name="TESTS_ZEND_LDAP_ALT_PASSWORD" value="" />
+        <env name="TESTS_ZEND_LDAP_ALT_PASSWORD" value="insecure" />
```

Comment:
I changed this line as an empty password is not working at all for login. So it's a bad default